construction_progress_management/__manifest__.py

```diff
@@ -0,0 +1,36 @@
+{
+    'name': 'Construction Progress Management',
+    'version': '1.0',
+    'category': 'Hidden/Tools',
+    'depends': [
+        'base_automation',
```

Comment:
Probably included in web_studio, no? We don't have this issue with standard modules
```suggestion
```